### DISCUSSION TITLE
section 6, when run make slither, //slither-disable-next-line not work

### DISCUSSION THREAD

**Discussion by Vincent71399** [Cyfrin/security-and-auditing-full-course-s23#240] [Q&A]
even with //slither-disable-next-line or //slither-disable-start, the issue will still appear in the report below. 

function getCalculatedFee(IERC20 token, uint256 amount) public view returns (uint256 fee) {
    //slither-disable-next-line divide-before-multiply
    uint256 valueOfBorrowedToken = (amount * getPriceInWeth(address(token))) / s_feePrecision;
    //slither-disable-next-line divide-before-multiply
    fee = (valueOfBorrowedToken * s_flashLoanFee) / s_feePrecision;
}

![image](https://github.com/user-attachments/assets/bff8e180-dc17-4636-955b-52df98911df6)

**Comment by EngrPips** [comment]
Hello @Vincent71399, It has been long since I used Slither, so I am unsure what is wrong here, but I think you can report on Slither's official GitHub page https://github.com/crytic/slither
